dts/arm/nuvoton/npcx.dtsi

```diff
@@ -691,6 +691,12 @@
 			label = "I2C_7_PORT_0";
 			status = "disabled";
 		};
+
+		power_ctrl_psl: power-ctrl-psl {
```

Comment:
ditto

Reply:
Please refer the board DT file [here](https://chromium-review.googlesource.com/c/chromiumos/platform/ec/+/3698178/3/zephyr/projects/npcx_evb/npcx9/gpio.dts) in the downstream repo.

Reply:
ok, fair enough